easybuild/easyconfigs/l/libGridXC/libGridXC-1.1.0-gompi-2022a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'libGridXC'
+version = '1.1.0'
+
+homepage = 'https://gitlab.com/siesta-project/libraries/libgridxc'
+description = """A library to compute the exchange and correlation energy
+ and potential in spherical (i.e. atoms) or periodic systems."""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'opt': True}
+
+source_urls = ['https://gitlab.com/siesta-project/libraries/libgridxc/-/archive/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e7883e57a4db2438ee59437740291c06e0cfe1c8ae1901e4001f32312307e46a']
+
+configopts = "-DWITH_MPI=ON -DWITH_LIBXC=ON"
```

Comment:
So, we can't do both mpi and non-mpi version of this anymore? I'll admit I'm not super familiar with this software or the stuff that uses it, perhaps the non-mpi versions was pointless.

Reply:
> So, we can't do both mpi and non-mpi version of this anymore? I'll admit I'm not super familiar with this software or the stuff that uses it, perhaps the non-mpi versions was pointless.

That seems to be a downside of the new cmake build system that they have implemented yes. As far as I could find in the installation instructions there no longer is an option to request a multiconfig build and it always builds the library with the same name (so redoing a second time with/without MPI would just overwrite the previous build). I personally never used the non-MPI version (and don't think there is a large use case for it), but nevertheless I could ask the developers if they know if there's still a way to have two versions?